### PR TITLE
Add objectSchema method to Realm Object

### PR DESCRIPTION
**Pull request by bigfish24**
Not sure if this is correct, since I couldn't get the tests to run.

### CODE REVIEW

**Comment by alazier** [comment]
You should use a method `objectSchema()` rather than a property.

File: tests/js/object-tests.js [hunk 1]
```diff
@@ -470,6 +470,14 @@ module.exports = {
             obj.doubleCol;
         });
     },
+    
+    testObjectSchema: function() {
+        var realm = new Realm({schema: [schemas.TestObject]});
+        realm.write(function() {
+            var obj = realm.create('TestObject', {doubleCol: 1});
+            TestCase.assertEqual(obj.objectSchema, schemas.TestObject);
```
Comment (alazier) [inline tests/js/object-tests.js]: Need to change this to a method.

**Comment by bigfish24** [comment]
@blagoev I can't get the tests to run and this fails CI... not sure what to do?

**Comment by blagoev** [comment]
@bigfish24 I will take a look why that is

**Comment by blagoev** [comment]
maybe we need to add it to the docs as well

**Pull request merged**